### PR TITLE
remove deprecations like RemovedInDjango110Warning

### PR DESCRIPTION
**Pull request by shimizukawa**
changes:

* remove RemovedInDjango110Warning

### CODE REVIEW

File: beproud/django/commons/models/fields.py [hunk 1]
```diff
@@ -1,18 +1,6 @@
...
-try:
-    import cPickle as pickle
-except ImportError:
-    import pickle
-
-try:
-    import json
-except ImportError:
-    import simplejson as json
-
```
Comment (shimizukawa) [inline beproud/django/commons/models/fields.py]: unused imports

File: beproud/django/commons/templatetags/common_tags.py [hunk 2]
```diff
@@ -1,128 +0,0 @@
...
-    against::
-
-        {% switch username %}
-            {% case "Jim" "Bob" "Joe" %}
-                Me old mate {{ username }}! How ya doin?
-            {% else %}
-                Hello {{ username }}
-        {% endswitch %}
-    """
-    warnings.warn('The switch template tag is deprecated. Use {% if ... %}...{% elif ... %}...{% endif %} (Django >1.4)')
```
Comment (shimizukawa) [inline beproud/django/commons/templatetags/common_tags.py]: `switch` tag was already deprecated since django-1.4 world.

File: tests/conftest.py [hunk 3]
```diff
@@ -1,4 +1,6 @@
 import os
+import warnings
+warnings.filterwarnings("error", module='beproud.django.commons')
```
Comment (shimizukawa) [inline tests/conftest.py]: All warnings include RemovedInDjango110Warning is be error.

**Comment by opapy** [comment]
LGTM!

**Pull request merged**